PowerEditor/src/winmain.cpp

```diff
@@ -609,7 +610,8 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE, PWSTR pCmdLine, int)
         }
 	}
 
-	Notepad_plus_Window notepad_plus_plus;
+	auto upNotepadWindow = std::make_unique<Notepad_plus_Window>();

```

Comment:
`up` in `upNotepadWindow` means `unique pointer` I suppose?

Reply:
Exactly, `up` in `upNotepadWindow` stands for "**u**nique **p**ointer".

Reply:
It's OK for "up" - I just want to make sure.